Assets/MixedRealityToolkit.Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -63,12 +67,39 @@ public MixedRealityCameraProfile CameraProfile
                 return cameraProfile;
             }
         }
+        
+        /// <inheritdoc/>
+        public Camera Main
+        {
+            get
+            {
+                if (main == null || !main.isActiveAndEnabled)
+                {
+                    FindOrCreateMainCamera();
+                }
+                return main;
+            }
+        }
 
         private DisplayType currentDisplayType;
         private bool cameraOpaqueLastFrame = false;
+        private static Camera main;
 
         /// <inheritdoc />
         public override void Initialize()
+        {
+            FindOrCreateMainCamera();
+
+            // Ensure that the camera is parented under the mixed reality playspace
+            main.transform.SetParent(MixedRealityPlayspace.Transform);
+
+            // There's lots of documented cases that if the camera doesn't start at 0,0,0, things break with the WMR SDK specifically.
+            // We'll enforce that here, then tracking can update it to the appropriate position later.
```

Comment:
If this is WMR SDK specific, this forced position should likely be contained in UNITY_WSA || UNITY_STANDALONE_WINDOWS stub.

Reply:
See below.